libraries/chain/include/eosio/chain/webassembly/eos-vm-oc/intrinsic_mapping.hpp

```diff
@@ -187,6 +187,9 @@ inline constexpr auto get_intrinsic_table() {
       "env.tapos_block_prefix",
       "env.tapos_block_num",
       "env.get_action",
+      "env.call",
+      "env.get_call_data",
+      "env.set_call_return_value",
```

Comment:
Don't these have to be added at the end?

Reply:
Thanks! Yes, they can only be added at the bottom. Wondering why none of OC tests in CICD fails.

Reply:
None of the tests have an existing code cache (populated with a version of nodeos prior to this change).